qemu/tests/cfg/virt_firmware_basic_test.cfg

```diff
@@ -0,0 +1,11 @@
+- virt_firmware_basic_test:
+    no i440fx
```

Comment:
Actually, only x86 and aarch64 can use edk2, so this is another limitation.

Reply:
Thanks Yihuang, I will update it.